packages/vc-verification/test/issuer-verification-test.ts

```diff
@@ -627,6 +627,78 @@ function testSuite() {
       ).to.be.true;
     });
 
+    it('RoleEIP191Jwt without claimType should not break verification,', async () => {
+      const adminJWT = new JWT(adminKeys);
+      const claimAdmin = {
+        claimData: { fields: {}, claimTypeVersion: 1 },
```

Comment:
@JGiter Is this the key line in the test then? This `claimData` is missing a `claimType`?

Reply:
yes, and this test breaks others